addons/payment_stripe/static/src/js/stripe_mixin.js

```diff
@@ -166,8 +164,24 @@ export default {
      *
      * @private
      * @param {object} processingValues - The processing values of the transaction.
+     * @param {number} paymentOptionId - The id of the payment option handling the transaction.
+     * @param {string} paymentMethodCode - The code of the selected payment method, if any.
      * @return {object} The processing error, if any.
      */
-    async _stripeConfirmIntent(processingValues) {},
+    async _stripeConfirmIntent(processingValues, paymentOptionId, paymentMethodCode) {
```

Comment:
It looks like it's not used here (at the moment) 
```suggestion
    async _stripeConfirmIntent(processingValues, paymentOptionId,) {
```